### ISSUE TITLE
Wrong symbol used in elementary_charge defintion

### ISSUE THREAD

**Issue by pfbuxton** [bpatel2107/pyrokinetics#317]
Hello,

The following line, https://github.com/pyro-kinetics/pyrokinetics/blob/unstable/src/pyrokinetics/units.py#L130, produced an error for me.

The minus sign is `U+2212`, which is a slightly longer negative symbol. Instead `U+002d`, which is the regular negative symbol, should be used.
```python
self.define("elementary_charge = 1.602176634e−19 coulomb")
```

In general, an alternative to defining these constants yourself you can either use:
* scipy's constants library
```python
from scipy import constants
elementary_charge = constants.elementary_charge
```
* or pint's inbuilt unit registry (probably better):
```python
import pint
elementary_charge = ureg('elementary_charge').to_preferred([ureg.coulomb])
```

**Comment by bpatel2107** [comment]
That is an incredible spot!

We actually do import that from scipy but don't use it from some reason.

https://github.com/pyro-kinetics/pyrokinetics/blob/ec1546b79fc0789019bb32b5a5927ec7d704472e/src/pyrokinetics/constants.py#L11

Same with the deuterium mass. Relatively easy to fix this!

**Comment by bpatel2107** [comment]
Out of curiosity what version of python are you using and on what operating system? I have not seen an issue with this before so would like know when this crops up.

**Comment by pfbuxton** [comment]
I am using python 3.12.

**Comment by bpatel2107** [comment]
Ah we don't test for python 3.12 so that may be the issue. In any case we'll make a PR for this

**Comment by bpatel2107** [comment]
@pfbuxton fixed in #318, will be in our next release but feel free to pull unstable now and use that
